tests/wpunit/Tribe/Tickets/Attendee_Info/ViewTest.php

```diff
@@ -1,73 +0,0 @@
-<?php
-
-namespace Tribe\Tickets\Attendee_Info;
```

Comment:
So this one is plan to be replaced or is no longer relevant?

Reply:
No longer relevant because we're not using the method that was being used at first (`template_redirect`). That test was checking if the script exited or not. We're now using `template_include`, hijacking the context, creating a fake page, stealing the content and replacing that with what we get from the templates. Ideally the test to check this, should be different. I've been discussing with Luca about it.

And this one is not relevant any more.